core/modules/user/user.module

```diff
@@ -511,7 +511,7 @@ function user_has_role($rid, $account = NULL) {
     $account = $GLOBALS['user'];
   }
 
-  return isset($account->roles[$rid]);
+  return in_array($rid, $account->roles);
```

Comment:
There are no `rid`s in Backdrop; this needs to be changed to role (machine) name instead (also update the docblock - perhaps also mention that `$account` refers to the account object?).

Reply:
Yeah I was going to do that but it was late last night so I just did it quick.

BTW, there is no account object. It already says `@param User $account` so nothing more to do.